listenbrainz/webserver/templates/user/import.html

```diff
@@ -58,4 +58,13 @@ <h3>Import a Last.fm backup file</h3>
       return false;
     }
   </script>
+  <script type="text/javascript">
+    function check() {
```

Comment:
check is a very generic name for this. check_import_button() maybe?

Reply:
Seems good. ``check_input()`` or ``check_username()`` maybe?